documentation/domains/Domain.json

```diff
@@ -569,8 +569,16 @@
     "Istio": {
       "type": "object",
       "properties": {
+        "replicationChannelPort": {
+          "description": "The operator will create a `T3` protocol WebLogic network access point on each WebLogic Server that is part of a cluster with this port to handle EJB and servlet session state replication traffic between servers. This setting is ignored for clusters where the WebLogic cluster configuration already defines a `replication-channel` attribute. Defaults to 4564.",
+          "type": "number"
+        },
+        "localhostBindingsEnabled": {
+          "description": "This setting was added in operator version 3.3.3, defaults to the `istioLocalhostBindingsEnabled` [Operator Helm value]({{\u003c relref \"/userguide/managing-operators/using-helm.md\" \u003e}}) which in turn defaults to `true`, and is ignored in version 4.0 and later. In version 3.x, when `true`, the operator creates a WebLogic network access point with a `localhost` binding for each existing channel and protocol.  In version 3.x, use `true` for Istio versions prior to 1.10 and set to `false` for version 1.10 and later.  Version 4.0 and later requires Istio 1.10 and later, will not create localhost bindings, and ignores this attribute.",
```

Comment:
The way I see it, you have two choices: one is remove the references to 4.0 and just describe the current situation with operator 3.x and the second is to keep it as written but add the caveat that version 4.0 is not yet available. With either choice, you will have to go back and edit the information when 4.0 becomes available. From a technical writing "best practices " POV, you should remove references to 4.0. We are instructed to almost never refer to releases/versions that exist only in the future.

Reply:
Instead of removing references to 4.0 or adding a qualifier sentence, maybe we tag all mentions of 4.0 with the word unreleased - e.g. something change "4.0" to something like "4.0 (unreleased)" or "4.beta"?

4.0 is technically available in that the full source is public in "main". 

This is going to keep happening, and it's hard to juggle fine distinctions between source and documentation, and even harder to maintain multiple versions of the same doc.